bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/dto/TapoRequest.java

```diff
@@ -24,7 +22,6 @@
  *
  * @author Christian Wild - Initial contribution
  */
-@NonNullByDefault
```

Comment:
Please revert

Reply:
I was able to provide the same comment.
Rather add @Nullable to the second param of the first constructor.